app/views/automation_manager/_form.html.haml

```diff
@@ -76,7 +78,7 @@
       = _("Credentials")
     = render :partial => "layouts/angular-bootstrap/auth_credentials_angular_bootstrap",
              :locals  => {:ng_show           => true,
-                          :ng_model          => "automationManagerModel",
+                          :ng_model          => "vm.automationManagerModel",
```

Comment:
This is wrong.. `ng_model` should stay the same here, but you can add `:vm_scope => true` to achieve the same result...

Then you can have `canValidateBasicInfo` on `vm` too :)